libethereum/State.cpp

```diff
@@ -243,9 +243,9 @@ unordered_map<Address, u256> State::addresses() const
 void State::setRoot(h256 const& _r)
 {
 	m_cache.clear();
-	m_touched.clear();
+//	m_touched.clear();
```

Comment:
yes - used when we want to change the root to an intermediate transaction in the block. still important to understand which addresses have changed during the block for `diff` to work.